packages/@pollyjs/adapter-node-http/tests/integration/adapter-test.js

```diff
@@ -125,26 +125,38 @@ function commonTests(transport) {
 
   it('should be able to download binary content', async function() {
     const url = `${protocol}//via.placeholder.com/150/92c952`;
-    const { server } = this.polly;
 
-    server.get(url).passthrough(true);
```

Comment:
We weren't able to catch this issue since we were doing a passthrough which still went through the adapter.